cpp/test/Slice/escape/Client.cpp

```diff
@@ -121,8 +121,8 @@ void testtypes(const Ice::CommunicatorPtr& communicator)
     _cpp_and::switchPtr f1 = std::make_shared<switchI>();
 
     _cpp_and::doPrxPtr g;
-    g->_cpp_case(0, d2);
-    g->_cpp_explicit();
+//    g->_cpp_case(0, d2);
+//    g->_cpp_explicit();
```

Comment:
We should open an issue for this so we don't forget.

Reply:
And we should not comment out this code as part of this PR.